jenny/templates/_helpers.tpl

```diff
@@ -0,0 +1,85 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "jenny.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "jenny.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "jenny.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "jenny.labels" -}}
+helm.sh/chart: {{ include "jenny.chart" . }}
+{{ include "jenny.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Selector labels
+*/}}
+{{- define "jenny.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "jenny.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+app: jenny
+service: app
+{{- end }}
+
+{{- define "jenny.envs" -}}
+env:
+  - name: "DJANGO_SETTINGS_MODULE"
+    value: "{{ .Values.django.settings }}"
+  - name: "DJANGO_CONFIGURATION"
+    value: "{{ .Values.django.configuration }}"
```

Comment:
We usually don't include the key where a template is used (for readability purpose when it is used I presume).

```suggestion
- name: "DJANGO_SETTINGS_MODULE"
  value: "{{ .Values.django.settings }}"
- name: "DJANGO_CONFIGURATION"
  value: "{{ .Values.django.configuration }}"
```